catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/IngestCommand.java

```diff
@@ -452,60 +491,144 @@ private void moveToFailedIngestDirectory(File source) {
+                LOGGER.error("No Zip Transformer found.  Unable to transform zip file into metacard list.");
+                INGEST_LOGGER.error("No Zip Transformer found.  Unable to transform zip file into metacard list.");
+            }
+
+        } else {
+
+            ingestStream.map(Path::toFile)
+                    .filter(file -> !file.isDirectory())
+                    .forEach(file -> {
+
+                        if (file.isHidden()) {
                             ignoreCount.incrementAndGet();
-                            printProgressAndFlush(start,
-                                    fileCount.get(),
-                                    ingestCount.get() + ignoreCount.get());
                         } else {
-                            Metacard result;
-                            try {
-                                result = readMetacard(file);
-                            } catch (IngestException e) {
-                                result = null;
-                                logIngestException(e, file);
-                                if (failedIngestDirectory != null) {
-                                    moveToFailedIngestDirectory(file);
-                                }
-                                printErrorMessage(
-                                        "Failed to ingest file [" + file.getAbsolutePath() + "].");
-                                if (INGEST_LOGGER.isWarnEnabled()) {
-                                    INGEST_LOGGER.warn("Failed to ingest file [{}].",
-                                            file.getAbsolutePath());
+                            String extension = "." + FilenameUtils.getExtension(file.getName());
+
+                            if (ignoreList != null && (ignoreList.contains(extension)
+                                    || ignoreList.contains(file.getName()))) {
+                                ignoreCount.incrementAndGet();
+                                printProgressAndFlush(start,
+                                        fileCount.get(),
+                                        ingestCount.get() + ignoreCount.get());
+                            } else {
+                                Metacard result;
+                                try {
+                                    result = readMetacard(file);
+                                } catch (IngestException e) {
+                                    result = null;
+                                    logIngestException(e, file);
+                                    if (failedIngestDirectory != null) {
+                                        moveToFailedIngestDirectory(file);
+                                    }
+                                    printErrorMessage(
+                                            "Failed to ingest file [" + file.getAbsolutePath()
+                                                    + "].");
+                                    if (INGEST_LOGGER.isWarnEnabled()) {
```

Comment:
I don't think it's necessary to check `isWarnEnabled()`: http://www.slf4j.org/faq.html#logging_performance